cms/api_test.py

```diff
@@ -134,3 +134,19 @@ def test_ensure_product_index(mocker):
     # Make sure the function is idempotent
     ensure_product_index()
     assert list(course_index_children_qset.all()) == [existing_course_page.page_ptr]
+    # make sure featured products are listing
+    HomeProductLink.objects.create(
+        page=home_page, course_product_page=existing_course_page
+    )
+    featured_products = home_page.products
+    assert len(featured_products) == 1
+    run = existing_course_page.product.first_unexpired_run
+    assert featured_products == [
+        {
+            "title": existing_course_page.title,
+            "description": existing_course_page.description,
+            "feature_image": existing_course_page.feature_image,
+            "start_date": run.start_date if run is not None else None,
+            "url_path": existing_course_page.get_url(),
+        }
+    ]
```

Comment:
@umarmughal824 can you add a test where we have more than 1 feature products and we assert the order of items on basis of soonest future date

Reply:
Let's wait for @pdpinch review, then will address the entire feedback altogether.

Reply:
What are "featured products" ?

Reply:
referring the term when adding product pages in `HomePage`